### ISSUE TITLE
Auto-pay for Lightning Invoice for small amount

### ISSUE THREAD

**Issue by shojofigure** [ZeusLN/zeus#1633]
### Describe the feature

Add an Auto-pay feature like WoS.

**Comment by myxmaster** [comment]
How does it work in Wallet of Satoshi?

**Comment by shojofigure** [comment]
https://github.com/ZeusLN/zeus/assets/134852413/4ae63d35-52f9-4aba-879f-b0498bdf036b

With Auto-pay off on WoS, when an invoice is pasted, it will bring up a Cancel / Send button option screen.


https://github.com/ZeusLN/zeus/assets/134852413/6883a783-fd8f-4a0f-b078-49dbb2979685

With Auto-pay ON, WoS will automatically Send once an invoice is pasted.

This is useful when zapping a lot on Nostr especially.

**Comment by shojofigure** [comment]
That's right, basically it removes the payment confirmation. WoS read the latest clipboard when opened, check for a valid lightning invoice, make the payment instantly.
